debezium-server/debezium-server-redis/pom.xml

```diff
@@ -69,7 +69,7 @@
         <dependency>
             <groupId>io.debezium</groupId>
             <artifactId>debezium-connector-postgres</artifactId>
-            <scope>test</scope>
+            <scope>runtime</scope>
```

Comment:
For the purpose of the integration test, we needed access to non-bean injected classes - in this case, the RedisOffsetStore. Quarkus seems to support this using the @IntegrationTest decorator, vs the @Test decorator. However, these integration tests only have access to the runtime scope (since they are simulating a real build). So the change of scope is needed for the Integration test to have access to the postgres connector.

Reply:
> in this case, the RedisOffsetStore

Couldn't you connect to it "out of bands"? Probably better anyways, so as to make sure the data is getting actually persisted in Redis.

Reply:
Couldn't find any documented way to get the Quarkus classloader to load anything that wasn't marked as 'runtime'. Let me know if there is a better way.